index.html

```diff
@@ -9,18 +9,32 @@
         <script src="js/fruits_and_veggies_test.js"></script>
         <script src="js/app.js"></script>
     </head>
-    <body>
+    <body ng-app="app" ng-controller="ctrl">
```

Comment:
Here is my banner for winning:

    <div ng-show="win" ng-class="{'has-success': win}">
        Congrats! You Won!
    </div>